src/docs/backup.dox

```diff
@@ -175,6 +175,16 @@ An example of opening the backup data source for block-based incremental backup:
 
 @snippet ex_all.c incremental block backup
 
+The URI \c backup:query_id can be used to return existing block incremental identifier
+strings. It operates like a backup cursor but will return the identifier strings
+as the keys of the cursor. There are no values. As with all backup cursors, there
+can only be one backup cursor of any type open at a time.
+
+An example of opening the backup data source to query incremental identifiers:
```

Comment:
Currently, the example of how to query existing IDs is only shown for the C language documentation. I feel moving this sentence of `An example` also into the m_if{c} would be good.